SimG4Core/PrintGeomInfo/test/python/g4OverlapCheck_cfg.py

```diff
@@ -18,7 +18,7 @@
 # Geant4 geometry check 
 process.g4SimHits.G4CheckOverlap.OutputBaseName = cms.string("cms2021")
 process.g4SimHits.G4CheckOverlap.OverlapFlag = cms.bool(True)
-process.g4SimHits.G4CheckOverlap.Tolerance  = cms.double(0.01)
+process.g4SimHits.G4CheckOverlap.Tolerance  = cms.double(0.001)
```

Comment:
Better not to do this, because log file is big enough even with 0.01

Reply:
Done that too